src/reanimated2/mappers.ts

```diff
@@ -142,11 +142,7 @@ function createMapperRegistry<T>() {
       }
     } else if (isSharedValue<T>(inputs)) {
       resultArray.push(inputs);
-    } else if (
-      inputs !== null &&
-      inputs !== undefined &&
-      isPlainJSObject(inputs)
-    ) {
+    } else if (inputs && isPlainJSObject(inputs)) {
```

Comment:
I've made it like that initially to do not throw away any other possible values, but actually we do not care here about any other [falsy](https://developer.mozilla.org/en-US/docs/Glossary/Falsy) values as they would no pass `isPlainJSObject` anyway.